### ISSUE TITLE
Vuex Store definition file not generated when manually created

### ISSUE THREAD

**Issue by mustaqimM** [quasarframework/quasar#6942]
**Describe the bug**
When manually running:
`quasar new store test`
The Vuex Store definition file: `index.js` file under `src/store` isn't being created.

**To Reproduce**
1. Without having a store, run: `quasar new store test`
2. The `index.js` file isn't created at `src/store/index.js`

**Expected behavior**
The `index.js` Vuex Store definition should be created when running `quasar new store test`.

**Additional context**
The `index.js` file being referenced:
```js
import Vue from 'vue'
import Vuex from 'vuex'

// import example from './module-example'

Vue.use(Vuex)

/*
 * If not building with SSR mode, you can
 * directly export the Store instantiation;
 *
 * The function below can be async too; either use
 * async/await or return a Promise which resolves
 * with the Store instance.
 */

export default function (/* { ssrContext } */) {
  const Store = new Vuex.Store({
    modules: {
      // example
    },

    // enable strict mode (adds overhead!)
    // for dev mode only
    strict: process.env.DEV
  })

  return Store
}
```

This file is created when first defining the project when `Vuex` is selected as an option.

**Comment by rstoenescu** [comment]
Will be available in "quasar" v1.8.6.

**Comment by mustaqimM** [comment]
thank you!
